_data/tools.yml

```diff
@@ -1505,3 +1505,14 @@
     existing OpenAPI specs.
   v2: true
   v3: true
+
+- name: Django REST Framework
+  category: server
```

Comment:
This is actually backwards of what we've currently got server defined as.

> Server Implementations: Easily create and implement resources and routes for your APIs.

We talk about using OpenAPI to make servers, but this tool uses code you've already written to retroactively create some OpenAPI that can then be used for... probably docs or something else. 

Maybe it could be docs, or annotations, or one of the other categories.

Reply:
@philsturgeon Which category do you suggest? Happy to change it.

It's not `Documentation: Render API Description as HTML (or maybe a PDF) so slightly less technical people can figure out how to work with the API.`

I don't see an annotations category.

There's always `Miscellaneous`...

Reply:
```suggestion
  category: miscellaneous
```